### PR TITLE
ci: use colima==0.5.6 on macOS

### PR DESCRIPTION
**Pull request by skshetry**
Hopefully this fixes the flaky tests.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -48,10 +48,14 @@ jobs:
       uses: pre-commit/action@v3.0.1
 
     # https://github.com/abiosoft/colima/issues/468
+    # colima v0.5.6 seems to run more stable than the latest - that has occasional network failures (ports are not open)
+    # see: https://github.com/abiosoft/colima/issues/962
     - name: Use colima as default docker host on MacOS
       if: matrix.os == 'macos-latest'
       run: |
-        brew install docker
+        brew install docker colima
```
Comment (skshetry) [inline .github/workflows/tests.yaml:56]:
```suggestion
        brew install docker
```

**Review by shcheklein — APPROVED** [review]
thanks @skshetry . Will we apply the same fix to DVCx?

**Comment by skshetry** [comment]
> thanks @skshetry . Will we apply the same fix to DVCx?

yes, I'll go through dvc plugins first.

**Comment by shcheklein** [comment]
@skshetry perfect! 🙏

**Pull request merged**